include/libtorrent/alert_types.hpp

```diff
@@ -2355,12 +2355,27 @@ namespace libtorrent
 		int m_size;
 	};
 
+	struct TORRENT_EXPORT dht_get_peers_reply_alert: alert {
+
+		dht_get_peers_reply_alert(aux::stack_allocator& alloc
+			, sha1_hash const& ih
+			, std::vector<tcp::endpoint> const& v);
+
+		const static int static_category = alert::dht_notification;
```

Comment:
do you think it would make sense to have a separate category for this alert? since it contains a lot more state and may be a bit more expensive.